src/software/ai/hl/stp/tactic/receiver_tactic.cpp

```diff
@@ -81,17 +81,17 @@ void ReceiverTactic::calculateNextIntent(IntentCoroutine::push_type& yield)
                                                       desired_angle, 0));
     }
 
-    // Check if we can shoot on the enemy goal from the receiver position
-    std::optional<std::pair<Point, Angle>> best_shot_opt =
-        calcBestShotOnEnemyGoal(field, friendly_team, enemy_team, *robot);
+    //    // Check if we can shoot on the enemy goal from the receiver position
```

Comment:
This variable was unused, can I remove this?

Reply:
Oops didn't see this comment, @garethellis0 @MathewMacDougall ?

Reply:
The commented-out one directly below this comment? Yeah, go ahead, thanks for checking!